packages/category-feed-luigis-box/src/Model/FeedItem/LuigisBoxCategoryFeedItemFactory.php

```diff
@@ -30,35 +29,31 @@ public function __construct(
     /**
      * @param \Shopsys\FrameworkBundle\Model\Category\Category $category
      * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
-     * @return \Shopsys\CategoryFeed\PersooBundle\Model\FeedItem\PersooCategoryFeedItem
+     * @return \Shopsys\CategoryFeed\LuigisBoxBundle\Model\FeedItem\LuigisBoxCategoryFeedItem
      */
-    public function create(Category $category, DomainConfig $domainConfig): PersooCategoryFeedItem
+    public function create(Category $category, DomainConfig $domainConfig): LuigisBoxCategoryFeedItem
     {
         $locale = $domainConfig->getLocale();
-        $hierarchyIds = [$category->getId()];
         $hierarchyNames = [$category->getName($locale)];
         $parent = $category->getParent();
         $rootCategory = $this->categoryRepository->getRootCategory();
 
         try {
-            $imageUrl = FeedItemImageHelper::limitWidthInImageUrl($this->imageFacade->getImageUrl($domainConfig, $category));
+            $imageUrl = ImageUrlWithSizeHelper::limitWidthInImageUrl($this->imageFacade->getImageUrl($domainConfig, $category));
         } catch (ImageNotFoundException) {
             $imageUrl = null;
         }
 
         while ($parent !== null && $parent->getId() !== $rootCategory->getId()) {
-            $hierarchyIds[] = $parent->getId();
             $hierarchyNames[] = $parent->getName($locale);
             $parent = $parent->getParent();
         }
 
-        return new PersooCategoryFeedItem(
+        return new LuigisBoxCategoryFeedItem(
             $category->getId(),
             $category->getName($locale),
-            array_reverse($hierarchyIds),
             $this->friendlyUrlFacade->getAbsoluteUrlByRouteNameAndEntityId($domainConfig->getId(), 'front_product_list', $category->getId()),
             array_reverse($hierarchyNames),
-            TransformString::convertHtmlToPlainText($category->getDescription($domainConfig->getId())),
```

Comment:
missing category description is also quite sad, they do not support it?

Reply:
Exactly